### DISCUSSION TITLE
Integrating a DTB deb file

### DISCUSSION THREAD

**Discussion by Austriker** [OE4T/meta-tegra#613] [Q&A] [answered]
Hi,

I would like to integrate a new carrier board but the supplier will not provide the DTS file, only the DTB file. Is it possible to integrate a deb DTB file with meta-tegra ? If possible how should I proceed ?

Best regards,

**Comment by madisongh** [comment, marked as answer]
Leaving aside the questionable nature of a supplier that won't turn over what should be copylefted source to a customer...

You might be able to make it work by writing a recipe to unpack and deploy the dtb file, setting KERNEL_DEVICETREE in your machine config to the name of the dtb file, and adding a bbappend to the linux-tegra recipe to unset KERNEL_DEVICETREE for your machine, so it doesn't try to compile the device tree from the nonexistent sources.
